Paco/src/com/pacoapp/paco/model/Event.java

```diff
@@ -117,8 +117,7 @@ public void writeToParcel(Parcel dest, int flags) {
     }
 
   }
-
-  @JsonIgnore
+  // Removing JsonIgnore so that we can enable coalescing of output records when we do getByQuery 
```

Comment:
This will create problems when uploading to the server because events have a series of ids on the server that are globally unique, not, just device-unique. 

I think we should drop the option not to coalesce. Then we don't have to worry about this. If we decide not to coalesce, it should just return raw rows, which would contain the id otherwise, we coalesce so that we don't need to manage this id problem.